pyproject.toml

```diff
@@ -49,14 +49,14 @@ python = "^3.7"
 
 poetry-core = "^1.3.2"
 poetry-plugin-export = "^1.1.2"
-"backports.cached-property" = { version = "^1.0.2", python = "<3.8" }
+"backports.cached-property" = { version = "^1.0.2", python = "3.7" }
 cachecontrol = { version = "^0.12.9", extras = ["filecache"] }
 cleo = "^1.0.0a5"
 crashtest = "^0.3.0"
 dulwich = "^0.20.46"
 filelock = "^3.8.0"
 html5lib = "^1.0"
-importlib-metadata = { version = "^4.4", python = "<3.10" }
+importlib-metadata = { version = "^4.4", python = "3.7" }
```

Comment:
Sure about that?

<html><body>
<!--StartFragment-->

importlib_metadata | stdlib
-- | --
4.8 | 3.11
4.4 | 3.10
1.4 | 3.8

<!--EndFragment-->
</body>
</html>
(from https://pypi.org/project/importlib-metadata/)

If we really want to do that, we'll have to change https://github.com/python-poetry/poetry/blob/0b71e4381445bf6b6c239bf0d88c8f5dfb4a32bd/src/poetry/utils/_compat.py#L10-L11